modules/collectcentBidAdapter.js

```diff
@@ -0,0 +1,94 @@
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER, NATIVE, VIDEO } from 'src/mediaTypes';
+import * as utils from 'src/utils';
+
+const BIDDER_CODE = 'collectcent';
+const URL = '//publishers.motionspots.com/?c=o&m=multi';
+const URL_SYNC = '//publishers.motionspots.com/?c=o&m=cookie';
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER, VIDEO, NATIVE],
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: (bid) => {
+    return Boolean(bid.bidId &&
+        bid.params &&
+        !isNaN(bid.params.placementId) &&
+        spec.supportedMediaTypes.indexOf(bid.params.traffic) !== -1
+    );
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {BidRequest[]} validBidRequests A non-empty list of valid bid requests that should be sent to the Server.
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: (validBidRequests, bidderRequest) => {
+    let winTop;
+    try {
+      winTop = window.top;
+      winTop.location.toString();
```

Comment:
For getting the page url, please use bidderRequest.refererInfo.referer
See: http://prebid.org/dev-docs/bidder-adaptor.html#referrers